packages/cli/config/tsconfig.json

```diff
@@ -6,6 +6,8 @@
     "noEmit": false,
     "declarationMap": true,
     "incremental": true,
-    "module": "es6"
+    "module": "es6",
+    "resolveJsonModule": true,
+    "esModuleInterop": true
```

Comment:
this change was needed to make the json fixture imports work. changing it just in the plugin doesn't work, for some reason. makes me think the plugin's tsconfig.json isn't being honored by backstage-cli?